### PR TITLE
E2Es on TEST for Most Watched Page Batch 2

### PR DESCRIPTION
**Pull request by LilyL0u**
**Overall change:**

E2Es enabled on test for the next batch of Most Watched pages. The URLs used are:

1. Afaan Oromoo - https://www.test.bbc.com/afaanoromoo/media/video
2. Afrique - https://www.test.bbc.com/afrique/media/video
3. Amharic - https://www.test.bbc.com/amharic/media/video
4. Arabic - https://www.test.bbc.com/arabic/media/video
5. Brasil - https://www.test.bbc.com/portuguese/media/video
6. Gahuza - https://www.test.bbc.com/gahuza/media/video
7. Igbo - https://www.test.bbc.com/igbo/media/video
8. Japanese - https://www.test.bbc.com/japanese/media/video
9. Korean - https://www.test.bbc.com/korean/media/video
10. Marathi - https://www.test.bbc.com/marathi/media/video
11. Nepali - https://www.test.bbc.com/nepali/media/video
12. Pashto - https://www.test.bbc.com/pashto/media/video
13. Persian - https://www.test.bbc.com/persian/media/video
14. Pidgin - https://www.test.bbc.com/pidgin/media/video
15. Punjabi - https://www.test.bbc.com/punjabi/media/video
16. Russian - https://www.test.bbc.com/russian/media/video
17. Somali - https://www.test.bbc.com/somali/media/video
18. Swahili - https://www.test.bbc.com/swahili/media/video
19. Tamil - https://www.test.bbc.com/tamil/media/video
20. Turkish - https://www.test.bbc.com/turkce/media/video
21. Vietnamese - https://www.test.bbc.com/vietnamese/media/video
22. Yoruba - https://www.test.bbc.com/yoruba/media/video

**Code changes:**
Enabled in settings file

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**
All pass on test

### CODE REVIEW

File: cypress/support/commands/application.js [hunk 1]
```diff
@@ -43,3 +43,42 @@ Cypress.Commands.add(
     );
   },
 );
+Cypress.Commands.add(
```
Comment (RichardPK) [inline cypress/support/commands/application.js]: Did you intend on these changes being in this PR Lily? Looks like the command is being used in onDemandRadio & onDemandTV (also included in this PR)

Comment (LilyL0u) [reply cypress/support/commands/application.js]: No I don't know how that got there as I didn't see it in VScode when I commited it... Sigh. Will remove.

File: cypress/integration/pages/onDemandRadio/tests.js [hunk 2]
```diff
@@ -43,7 +43,7 @@ export default ({ service, pageType, variant, isAmp }) => {
             const iframeURL = isBrandPage ? iframe.prop('src') : embedUrl;
 
             cy.get(`iframe[src*="${iframeURL}"]`).should('be.visible');
-            cy.testResponseCodeAndType(iframeURL, 200, 'text/html');
+            cy.testResponseCodeAndTypeRetry(iframeURL, 200, 'text/html');
```
Comment (RichardPK) [inline cypress/integration/pages/onDemandRadio/tests.js]: Do: `git checkout origin/latest cypress/integration/pages/onDemandRadio/tests.js`

File: cypress/integration/pages/onDemandTV/tests.js [hunk 3]
```diff
@@ -29,7 +29,7 @@ export default ({ service, pageType, variant, isAmp }) => {
             const iframeURL = isBrandPage ? iframe.prop('src') : embedUrl;
 
             cy.get(`iframe[src*="${iframeURL}"]`).should('be.visible');
-            cy.testResponseCodeAndType(iframeURL, 200, 'text/html');
+            cy.testResponseCodeAndTypeRetry(iframeURL, 200, 'text/html');
```
Comment (RichardPK) [inline cypress/integration/pages/onDemandTV/tests.js]: Do: `git checkout origin/latest cypress/integration/pages/onDemandTV/tests.js`

File: cypress/support/commands/application.js (hunk 1 above)
Comment (RichardPK) [reply cypress/support/commands/application.js]: Do: `git checkout origin/latest cypress/support/commands/application.js`

**Review by RichardPK — APPROVED** [review]

**Review by jroebu14 — APPROVED** [review]

**Pull request merged**